src/tests/simulation/tests/integration_tests/scale/cascade_materialized_view.rs

```diff
@@ -36,6 +37,16 @@ async fn test_simple_cascade_materialized_view() -> Result<()> {
     session.run(ROOT_TABLE_CREATE).await?;
     session.run(MV1).await?;
 
+    // Check internal progress
+    let internal_table = session.run(SHOW_INTERNAL_TABLES).await?;
```

Comment:
Would you mind moving these codes into a new test case? Seems not related to the original test.